test-network-function/platform/suite.go

```diff
@@ -696,7 +696,7 @@ func testNodeHugepagesWithMcSystemd(nodeName string, nodeNumaHugePages, mcSystem
 				}
 			}
 			if !configMatching {
-				return false, fmt.Errorf(fmt.Sprintf("MC numa=%d, hugepages (count:%d, size:%d) not matching node ones: %s",
+				return false, fmt.Errorf("%s", fmt.Sprintf("MC numa=%d, hugepages (count:%d, size:%d) not matching node ones: %s",
```

Comment:
Errorf + Sprintf seems redundant here... my fault, this was my code. But how come the linter did/does not complain? This can be left as:
```
return false, fmt.Errorf("MC numa=%d, hugepages (count:%d, size:%d) not matching node ones: %s",
    mcNumaIdx, mcHugepagesCfg.hugepagesCount, mcHugepagesCfg.hugepagesSize, nodeNumaHugePages)
```

Reply:
Yeah I thought so too but the linter was only complaining about the %s.  I will updated.